bands/templates/band_admin/index.html

```diff
@@ -13,46 +13,52 @@
   <table id="group-table" class="datatable table table-striped" data-page-length="-1">
     <thead>
         <tr>
             <th>Group</th>
-            {% if c.BAND_PANEL_DEADLINE %}<th>Panel</th>{% endif %}
-            {% if c.BAND_BIO_DEADLINE %}<th>Bio Provided</th>{% endif %}
-            {% if c.BAND_AGREEMENT_DEADLINE %}<th>Agreement Completed</th>{% endif %}
-            {% if c.BAND_W9_DEADLINE %}<th>W9 Uploaded</th>{% endif %}
-            {% if c.BAND_MERCH_DEADLINE %}<th>Merch</th>{% endif %}
-            {% if c.BAND_CHARITY_DEADLINE %}<th>Charity</th>{% endif %}
-            {% if c.BAND_BADGE_DEADLINE %}<th>Badges Claimed</th>{% endif %}
-            {% if c.STAGE_AGREEMENT_DEADLINE %}<th>Stage Plans</th>{% endif %}
-            <th>Band?</th>
+            {% for item in c.CHECKLIST_ITEMS %}
+              <th>{{ item.header }}</th>
+            {% endfor %}
+            <th>Group Type</th>
         </tr>
     </thead>
     <tbody>
     {% for group in groups %}
-        <tr id="{{ group.id }}" class="{{ group.band|yesno("band,nonband") }}">
+        <tr id="{{ group.id }}" class="{{ group.band.group_type_label|lower if group.band else "" }} group-row">
             <td>
               <a href="../groups/form?id={{ group.id }}">{{ group.name }}</a>
-              <span class="band_info">{% if group.band %}(<a href="band_info?id={{ group.band.id }}" class="text-nowrap">Band Info</a>){% endif %}</span>
+              <span class="band_info">
+                {% if group.band %}
+                  (<a href="group_info?id={{ group.band.id }}" class="text-nowrap">{{ group.band.group_type_label }} Info</a>)
+                {% endif %}</span>
             </td>
-            {% if c.BAND_PANEL_DEADLINE %}<td>{{ group.band.panel.status if group.band else '' }}</td>{% endif %}
-            {% if c.BAND_BIO_DEADLINE %}<td>{{ group.band.bio.status if group.band else '' }}</td>{% endif %}
-            {% if c.BAND_AGREEMENT_DEADLINE %}<td>{{ group.band.info.status if group.band else '' }}</td>{% endif %}
-            {% if c.BAND_W9_DEADLINE %}<td>
-                {% if group.band and not group.band.payment %}Not Needed
-                {% else %}{{ group.band.taxes.status|url_to_link("Yes", "_blank") if group.band else '' }}{% endif %}</td>{% endif %}
-            {% if c.BAND_MERCH_DEADLINE %}<td>{{ group.band.merch.status if group.band else '' }}</td>{% endif %}
-            {% if c.BAND_CHARITY_DEADLINE %}<td>{{ group.band.charity.status if group.band else '' }}</td>{% endif %}
-            {% if c.BAND_BADGE_DEADLINE %}<td>{{ group.unregistered_badges ~ " Unclaimed" if group.unregistered_badges else "Yes" }}</td>{% endif %}
-            {% if c.STAGE_AGREEMENT_DEADLINE %}<td>{{ group.band.stage_plot.status|url_to_link("Yes", "_blank") if group.band else '' }}</td>{% endif %}
+            {% for item in c.CHECKLIST_ITEMS %}
+              {% set item_display = group.band.status(item.name) if group.band else '' %}
```

Comment:
Note to self: I may have broken this by converting an in-template dict to a global dict later on. Check that it is `item.name` and not `item['name']`.